dotfiles/prompt

```diff
@@ -160,3 +160,6 @@ _prompt() {
 }
 
 PROMPT_COMMAND='_prompt'
+if [[ "$TERM_PROGRAM" = Apple_Terminal ]]; then
+  export PROMPT_COMMAND="$PROMPT_COMMAND;update_terminal_cwd;"
```

Comment:
I don't think so; that's why I added [L163][1], since weirdly iTerm doesn't have the command. So it should only trigger if you're using the Apple Terminal.

[1]: https://github.com/bendyworks/buffet/pull/25/files#diff-e3d9e17e1e21c3229816ebb10df99994R163